setup.cfg

```diff
@@ -1,2 +1,2 @@
 [flake8]
-max-line-length = 95
+max-line-length = 130
```

Comment:
we kept exceeding 95, so this quiets my linter

Reply:
Oh, my bad.  I have my ide setting set to 120